CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 <a name="Pending Release"></a>
 ## [Pending Release](https://github.com/lightstep/otel-launcher-java/compare/0.10.1...master)
+* Add Metrics support
+* Use OpenTelemetry Java API version 0.11.0
+* Use OpenTelemetry Java Instrumentation version 0.11.0
```

Comment:
Also add that we are now using `b3multi` as the default propagator?

Reply:
added